data/gegno/gegno outfits.txt

```diff
@@ -789,6 +789,90 @@ effect "drilling"
 	"velocity scale" -0.01
 
 
+outfit "Burrower"
+	category "Guns"
+	licenses
+		"Gegno Civilian"
+		"Gegno Driller"
+	cost 326000
+	"thumbnail" "outfit/burrower"
+	"mass" 32
+	"outfit space" -28
+	"weapon capacity" -28
+	"gun ports" -1
+	"drill spar" -1
+	weapon
+		sound "drill"
+		"hardpoint sprite" "hardpoint/burrower"
+		"hardpoint offset" 14
+		"hit effect" "burrow"
+		"inaccuracy" 0.4
+		"velocity" 30
+		"lifetime" 1
+		"reload" 1
+		"range override" 60
+		"velocity override" 4
+		"firing energy" 1.8
+		"firing heat" 24
+		"shield damage" .4
+		"hull damage" 6
+		"minable damage" 2
+		"hit force" 4
+	description `Burrower Drills are longer, thinner, and more precise than the larger Excavator Drill system, and although are nowhere near as powerful, can mine asteroids more carefully, preserving more materials in the process. Rather than constructing the spaceship around the base system, these drills are more compact and are mounted on self-sustaining drill spars, resulting in lower required internal space and machinery.`
```

Comment:
```suggestion
	description `Burrower Drills are longer, thinner, and more precise than the larger Excavator Drill system; although they are nowhere near as powerful, Burrowers can mine asteroids more carefully, preserving more materials in the process. Rather than constructing the spaceship around the base system, these drills are more compact and are mounted on self-sustaining drill spars, resulting in lower required internal space and machinery.`
```